### DISCUSSION TITLE
All of my work gets deleted, if accidently page gets reloaded

### DISCUSSION THREAD

**Discussion by priyanshk20** [remarkjs/remark#766] [Q&A] [answered]
Hi! How can I solve this issue, if a user uses my app , if by any chance their page gets refreshed all of their work is gone. How can I implement a feature to solve this issue?

Thanks.

**Comment by Murderlon** [comment, marked as answer]
Hi, we are going to need more information. See here how to [ask quality questions](https://github.com/unifiedjs/.github/blob/main/support.md#asking-quality-questions).

**Reply by ChristianMurphy** [reply]
To add on, this question seems to be specific to an editing experience.
If so, this would be a question for the editing software itself.
